src/strategies/spaceid/index.ts

```diff
@@ -0,0 +1,236 @@
+        (delegationsReverse[delegation.delegator] = delegation.delegate)
+    );
+  return Object.fromEntries(
+    addresses.map((address) => [
+      address,
+      Object.entries(delegationsReverse)
+        .filter(([, delegate]) => address.toLowerCase() === delegate)
+        .map(([delegator]) => getAddress(delegator))
+    ])
+  );
+}
+
+async function getDelegationPower(network, provider, addresses, snapshot) {
+  const strategies = [
+    {
+      name: 'erc20-balance-of',
+      params: {
+        address: '0x2dfF88A56767223A5529eA5960Da7A3F5f766406',
+        decimals: 18
+      }
+    }
+  ];
+  const delegationSpace = SPACE_NAME;
+  const delegations = await getDelegations(
+    delegationSpace,
+    network,
+    addresses,
+    snapshot
+  );
+  if (Object.keys(delegations).length === 0) return {};
+
+  const scores = (
+    await getScoresDirect(
+      SPACE_NAME,
+      strategies,
+      network,
+      provider,
+      Object.values(delegations).reduce((a: string[], b: string[]) =>
+        a.concat(b)
+      ),
+      snapshot
+    )
+  ).filter((score) => Object.keys(score).length !== 0);
+
+  return Object.fromEntries(
+    addresses.map((address) => {
+      const addressScore = delegations[address]
+        ? delegations[address].reduce(
+            (a, b) => a + scores.reduce((x, y) => (y[b] ? x + y[b] : x), 0),
+            0
+          )
+        : 0;
+      return [address, addressScore];
+    })
+  );
+}
+
+async function getLpTokenOnBsc(addresses, snapshot) {
+  const params = {
+    accounts: {
+      __args: {
+        where: {
+          id_in: addresses
+        },
+        block: snapshot !== 'latest' ? { number: snapshot } : { number_gte: 0 }
+      },
+      id: true,
+      withdraws: {
+        __args: {
+          where: { pool: PANCAKE_ID_USDT_PAIR }
+        },
+        inputTokenAmounts: true,
+        timestamp: true
+      },
+      deposits: {
+        __args: {
+          where: { pool: PANCAKE_ID_USDT_PAIR }
+        },
+        inputTokenAmounts: true,
+        timestamp: true
+      }
+    }
+  };
+
+  const pools = await subgraphRequest(pancakeV3Subgraph, params);
+
+  const pancakeIDLPScore = {};
+  for (const account of pools.accounts) {
+    let IdLPToken: BigNumber = BigNumber.from(0);
+    for (const withdraw of account.withdraws) {
+      IdLPToken = IdLPToken.add(BigNumber.from(withdraw.inputTokenAmounts[0]));
+    }
+    for (const deposit of account.deposits) {
+      IdLPToken = IdLPToken.sub(BigNumber.from(deposit.inputTokenAmounts[0]));
+    }
+    pancakeIDLPScore[account.id] = IdLPToken.div(WeiPerEther).toNumber();
+    pancakeIDLPScore[account.id] < 0 && (pancakeIDLPScore[account.id] = 0);
+  }
+  return pancakeIDLPScore;
+}
+
+export async function strategy(
+  space,
+  network,
+  provider,
+  addresses,
+  options,
+  snapshot
+) {
+  const delegationPower = await getDelegationPower(
+    network,
+    provider,
+    addresses,
+    snapshot
+  );
```

Comment:
Hi @SID-Marcus We are not accepting any new strategies if voting power depend on other voters

Reply:
Refer https://github.com/snapshot-labs/snapshot-strategies#code

Reply:
The method is almost as same as  DELEGATION  strategy. We made some changes on getting delegation data from subgraph because the official one is too slow on querying. Delegation power is a necessary part of voting power in our space, so I would appreciate it if you could approve it.

Reply:
After our discussion, we decided to remove this part of this strategy. Instead, we will use API strategy to get delegation power.

Reply:
After our discussion, we decided to remove this part of this strategy. Instead, we will use API strategy to get delegation power.